backend/.cargo/config.toml

```diff
@@ -0,0 +1,10 @@
+[build]
+rustflags = [
+    "--cfg",
+    "tokio_unstable",
+    "-C",
+    "link-arg=-fuse-ld=lld",
+    "-Clink-arg=-Wl,--no-rosegment",
```

Comment:
What's the benefit of this ?

Reply:
It passes --no-rosegment through to lld.
Which is documented as
> Do not put read-only non-executable sections in their own segment

It helps with stack traces of release builds and is necessary for some profiles.
I can remove it, but it doesn't hurt performance from what I know, it's only a size optimization